### ISSUE TITLE
Default rc file / init.txt

### ISSUE THREAD

**Issue by yrmvgh** [jeremygurr/dcssca#584]
I've been looking in old-style (sandman, whitepho) and new style (old-gorgo, old-myself, plus the new one) RC files.

The most common feature in RC files is the "set skill target" function, written by Elliptic & Gammafunk, who said: "We've talked about adding it into the default lua somehow, but it still has some issues menu-wise and probably should get a better in-game treatment."

So yes, it's a little bit clunky, but for the most part works really well.

I just observed a good player playing without the relevant LUA function, and they were popping open their skill menu every kill or two, especially when at x.8 or x.9 skill, to keep an eye on things and manually do the stuff.

So I say, let's just throw it in there. In the unlikely event someone doesn't want to use it, they can just set the target impossibly low (0) or high, and it'll never bother them again. And, conveniently, it opens the skill menu on new game as well, which everyone does.

**Comment by jeremygurr** [comment]
good idea. Should be built in functionality. You can choose not to set it, and it will behave like normal. Or you can set it in the skill menu.

**Comment by yrmvgh** [comment]
Will it work if I just throw it into initfile.cc somewhere near the end, or should it go in a different file?

**Comment by jeremygurr** [comment]
no it needs to just be built into the code. Lua should not be used any more than necessary. It's slow, clunky, and very difficult to debug.

**Comment by sooheon** [comment]
I'm curious, for both CA and vanilla, do all skills have breakpoints at n.0? None of the fractional skill levels help at all? Sounds like a lot of investment for nothing from 26.1 to 26.9 then.

**Comment by jeremygurr** [comment]
0.1 skill increments should always be meaningful. I'm nit aware if any that aren't. If you find one, tell me and I'll fix it.
